indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -285,26 +285,48 @@ private void extractSubjects(Element mods, IndexDocumentBean idb) {
 
     private void extractOtherSubjects(Element mods, IndexDocumentBean idb) {
         List<Element> otherSubjectEls = mods.getChildren("subject", JDOMNamespaceUtil.MODS_V3_NS);
-        List<String> otherSubjects = new ArrayList<>();
+        List<String> otherSubjects = extractNestedSubjects(otherSubjectEls, new ArrayList<>());
```

Comment:
This method doesn't really need a return value since you are modifying otherSubjects in place. So I'd suggest initializing otherSubjects to an empty array on the previous line and then calling the method on the next line, and removing the return type from the method.